prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-care-types-page/health-auth-care-types-page.component.ts

```diff
@@ -52,20 +60,33 @@ export class HealthAuthCareTypesPageComponent implements OnInit {
 
   public onSubmit() {
     if (this.formUtilsService.checkValidity(this.form)) {
-      const careTypes = [...new Set(this.careTypes.value.map(({ careType }) => careType.trim()) as string[])];
+      const careTypes = [...new Set(this.careTypes.getRawValue()
```

Comment:
Currently this doesn't work when you add a new custom care type, you will need some logic in here to handle the custom care types that are just a string
``` careType.name ? careType.name.trim() : careType.trim() ```

Reply:
Fixed